### PR TITLE
Clarifying some steps

### PR DESCRIPTION
**Pull request by albertmena**
Not sure abput how it looks in HTML format

### CODE REVIEW

File: docs/developer/tutorials/creating-streaming-protocol.rst [hunk 1]
```diff
@@ -363,8 +372,8 @@ Let's get our hands dirty..
                     # append it to registeredFiles list
 
                     # create a new step to register the new file
-                    # use _insertFunctionStep with registerImageStep, file, and
-                    # self.readXmlFile as a prerequisite parameter
+                    # use _insertFunctionStep with registerImageStepadding the parameter file, and
```
Comment (pconesa) [inline docs/developer/tutorials/creating-streaming-protocol.rst]: there is a missing space?

**Review by pconesa — CHANGES_REQUESTED** [review]
Thank you so much for reviewing and updating the documentation!

File: docs/developer/tutorials/creating-streaming-protocol.rst (hunk 1 above)
Comment (albertmena) [reply docs/developer/tutorials/creating-streaming-protocol.rst]: y e s

**Review by pconesa — APPROVED** [review]
Nice!

**Pull request merged**